src/app/components/pie-chart/pie-chart.component.html

```diff
@@ -0,0 +1,13 @@
+<div style="display: block">
+    <canvas baseChart
+            [data]="co2PieChartData"
+            [labels]="pieChartLabels"
+            [chartType]="pieChartType"
+            [colors]="pieChartColors"
+            [legend]="pieChartLegend"
+            (chartHover)="chartHovered($event)"
+            (chartClick)="chartClicked($event)">
+        </canvas>
+  </div>
+<!-- [data]="pieChartData" -->
+<!-- "{{co2PieChartData}}" -->
```

Comment:
Guess we should not commit commented code

Reply:
yeah sure. this is still work in progress!! will remove comments before creating a "serious" pull request :)